projects/scion/workbench/src/lib/content-projection/content-projection.directive.ts

```diff
@@ -85,7 +92,7 @@ export class ContentProjectionDirective implements OnInit, OnDestroy {
   }
 
   private setVisible(visible: boolean): void {
-    this.styleContent({display: visible ? null : 'none'});
+    this.styleContent({visibility: visible ? null : 'hidden'});
```

Comment:
Same here, describe why we are using 'visibility:hidden' over 'display:none'

```ts
    // We use `visibility: hidden` over `display: none` to preserve the dimension of projected content, crucial,
    // for example, if projected content implements virtual scrolling. This is because "display:none" sets width
    // and height to 0.
    this.styleContent({display: visible ? null : 'none'});
```

Reply:
done